### ISSUE TITLE
Twitch Recover keeps crashing

### ISSUE THREAD

**Issue by Kwquinch** [TwitchRecover/TwitchRecover#599]
Since Twitch Recover keeps crashing could someone give me the m3u8 link of this Vod https://twitchtracker.com/eliasn97/streams/40342998949

**Comment by expectaccesviolation** [comment]
https://dgeft87wbj63p.cloudfront.net/ef248cabe0b8349eeee0_eliasn97_40342998949_1704209500/chunked/index-dvr.m3u8

**Comment by Kwquinch** [comment]
> https://dgeft87wbj63p.cloudfront.net/ef248cabe0b8349eeee0_eliasn97_40342998949_1704209500/chunked/index-dvr.m3u8

How did you get the link?

**Comment by rorshek** [comment]
It would be nice if  [expectaccesviolation](https://github.com/expectaccesviolation) could tell you how he did it.

**Comment by expectaccesviolation** [comment]
@Kwquinch @rorshek i used [VodRecoverry](https://github.com/Shishkebaboo/VodRecovery)

**Comment by rorshek** [comment]
[expectaccesviolation](https://github.com/expectaccesviolation) I was able to restore it only through a link from the Sullygnome site, via Twitchtracker and Streamscharts i got 403 error, is that the same for you?

**Comment by expectaccesviolation** [comment]
yeah my bad shouldve told you that before, twitchtracker and streamcharts return an error currently

**Comment by rorshek** [comment]
[expectaccesviolation](https://github.com/expectaccesviolation) thank you, I will use this method for now

**Comment by BSClient** [comment]
> @Kwquinch @rorshekużyłem [VodRecoverry](https://github.com/Shishkebaboo/VodRecovery)

You will record a video with a guide on how to create a link in m3u8 format in VodRecoverry.

Because I don't know how to do it, how to install it and make a link in m3u8.

Please

**Comment by BSClient** [comment]
> @Kwquinch @rorshek i used [VodRecoverry](https://github.com/Shishkebaboo/VodRecovery)

You will record a video with a guide on how to create a link in m3u8 format in VodRecoverry.

Because I don't know how to do it, how to install it and make a link in m3u8.

Please

**Comment by BSClient** [comment]
> @expectaccesviolation @Kwquinch @rorshek i used [VodRecoverry](https://github.com/Shishkebaboo/VodRecovery)


You will record a video with a guide on how to create a link in m3u8 format in VodRecoverry.

Because I don't know how to do it, how to install it and make a link in m3u8.

Please

**Comment by expectaccesviolation** [comment]
@BSClient whats your discord?

**Comment by BSClient** [comment]
> @BSClient whats your discord?

mpgaming

**Comment by BSClient** [comment]
Thank you for your help <3
